containers/nova/Dockerfile.nova

```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1
+
+ARG OPENSTACK_VERSION="required_argument"
+FROM docker.io/openstackhelm/nova:${OPENSTACK_VERSION}-ubuntu_jammy
```

Comment:
These patches are likely quite specific to that version of the code, should we hardcode the base image here? Or just leave it and fix it when it breaks? Dunno.

Reply:
Yeah. I'd like to get to switching to building our own container from a branch we point to.